llvm/compiler-course/backend/savchenko_m_l3_v2/savchenko_m_l3_v2.cpp

```diff
@@ -0,0 +1,101 @@
+#include "X86.h"
+#include "X86InstrInfo.h"
```

Comment:
Have you tried to include the following 2 headers?
```suggestion
#include "X86InstrInfo.h"
#include "X86RegisterBankInfo.h"
#include "X86RegisterInfo.h"
```

Reply:
I tried to include these 2 headers, but when I try to build, I get an error stating that the header is `X86RegisterBankInfo.h` not found:
```bash
fatal error: 'X86RegisterBankInfo.h' file not found
    3 | #include "X86RegisterBankInfo.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
```

If I do not use `X86RegisterBankInfo.h`, an error is returned stating that the object `X86::VECRRegBankID` does not exist:
```bash
error: no member named 'VECRRegBankID' in namespace 'llvm::X86'
   68 |             if (RB && RB->getID() == X86::VECRRegBankID) {
      |                                      ~~~~~^
1 error generated.
ninja: build stopped: subcommand failed.
```